leases/schema/utils.py

```diff
@@ -47,8 +47,12 @@ def terminate_lease(
         info, lease_id, only_type=lease_type, nullable=False,
     )
 
-    if lease.status != LeaseStatus.PAID:
-        raise VenepaikkaGraphQLError(_(f"Lease is not paid: {lease.status}"))
+    if lease.status not in (LeaseStatus.PAID, LeaseStatus.ERROR):
```

Comment:
Can you add it to `leases/consts.py` so we have a single source of truth?

```python
TERMINABLE_STATUSES = (LeaseStatus.PAID, LeaseStatus.ERROR)
```

Reply:
good idea!